samples/unity-of-bugs/Assets/Resources/Sentry/SentryOptions.asset

```diff
@@ -9,12 +9,12 @@ MonoBehaviour:
   m_GameObject: {fileID: 0}
   m_Enabled: 1
   m_EditorHideFlags: 0
-  m_Script: {fileID: -158719865, guid: 90e21e4c8acb64ec1984c991c35727b0, type: 3}
+  m_Script: {fileID: -158719865, guid: 33c5c1e013e193f4296b0bb556af7f66, type: 3}
   m_Name: SentryOptions
   m_EditorClassIdentifier: 
   <Enabled>k__BackingField: 1
   <CaptureInEditor>k__BackingField: 1
-  <Dsn>k__BackingField: https://94677106febe46b88b9b9ae5efd18a00@o447951.ingest.sentry.io/5439417
+  <Dsn>k__BackingField: https://b8fd848b31444e80aa102e96d2a6a648@o510466.ingest.sentry.io/5606182
```

Comment:
This DSN is of the sentry-sdks unity project you should have access to:

https://sentry.io/organizations/sentry-sdks/issues/?project=5439417

```suggestion
  <Dsn>k__BackingField: https://94677106febe46b88b9b9ae5efd18a00@o447951.ingest.sentry.io/5439417
```